tests/ticker.py

```diff
@@ -798,7 +814,7 @@ def test_info(self):
 def suite():
     suite = unittest.TestSuite()
     suite.addTest(TestTicker('Test ticker'))
-    suite.addTest(TestTickerEarnings('Test earnings'))
+    #suite.addTest(TestTickerEarnings('Test earnings'))
```

Comment:
why comment?

Reply:
I branched from the wrong base I guess. Reverted.